scripts/verify-mdlint.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+# vim: set ts=4 sw=4 et :
+
+# Usage: pre-commit.sh [--require-all]
+#   --require-all  Fail instead of warn if a checker is not found
+
+set -e
+
+# Run checks from root of the repo
+scriptdir="$(dirname "$(realpath "$0")")"
+cd "$scriptdir/.."
+
+# run_check <file_regex> <checker_exe> [optional args to checker...]
+# Pass empty regex when no regex is needed
+function run_check() {
+    regex="$1"
+    shift
+    exe="$1"
+    shift
+
+    if [ -x "$(command -v "$exe")" ]; then
+      if [ -z "$regex" ]; then
+          "$exe" "$@"
+      else
+          find . -path ./vendor -prune -o -regextype egrep -iregex "$regex" -print0 |
+              xargs -0rt -n1 "$exe" "$@"
+      fi
+    elif [ "$all_required" -eq 0 ]; then
+        echo "Warning: $exe not found... skipping some tests."
+    else
+        echo "FAILED: All checks required, but $exe not found!"
+        exit 1
+    fi
+}
+
+all_required=0
```

Comment:
this is not required as you are not running multiple linter with `run_check`

Reply:
the plan is to have more linter check-in a follow-up PR.

Reply:
But they wouldn't run in this script, right?

`build.make` has a mechanism for running multiple tests, see "make test". Use that instead of writing one script which runs multiple unrelated checks.